meinberlin/templates/meinberlin_dashboard2/includes/project_list_item.html

```diff
@@ -57,8 +57,7 @@ <h2 class="list-item__title">
                                 <i class="fa fa-caret-down" aria-label="{% trans 'more' %}"></i>
                             </button>
                             {% if not project|is_external %}
-                            <ul class="dropdown-menu" aria-labelledby="project-actions">
-                                <li>
+                            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="project-actions">
                                     <form method="post" class="list-item__action">
```

Comment:
regressions :(
using `list-item__action` without an enclosing `list-item` is wrong if i understood bem correctly
@fuzzylogic2000 what do you think

Reply:
I think you're right, but luckily there is one in line 5. :)